docs/operators/kafkaInputOperator.md

```diff
@@ -280,3 +282,145 @@ Below is the configuration for “test” Kafka topic name and
 <value>localhost:2181</value>
 </property>
 ```
+
+
+### 0.9 Version of Kafka Input Operator
+
+### AbstractKafkaInputOperator (Package: org.apache.apex.malhar.kafka)
+
+This version uses the new 0.9 version of consumer API and features of this version are described here. This operator is fault-tolerant, scalable, multi-cluster and multi-topic support.
+
+#### Pre-requisites
+
+This operator requires version 0.9.0 or later of the Kafka Consumer API.
```

Comment:
I think this document should explain what the ready to use operators are. AbstractXXX is for implementing custom operator.